src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25433.cs

```diff
@@ -0,0 +1,26 @@
+﻿#if !MACCATALYST || !WINDOWS
```

Comment:
VerifyScreenshot is not implemented on Catalyst yet. Should not be necessary to avoid in that platform.
Why avoid it on Windows?

Reply:
Could you include a comment in code with a reference to the issue https://github.com/dotnet/maui/issues/11320?

Reply:
@jsuarezruiz, I have added a comment. Could you please check and provide your concerns if any?